docs/source/en/tutorials/using_peft_for_inference.md

```diff
@@ -133,6 +133,73 @@ image
 
 ![no-lora](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/diffusers/peft_integration/diffusers_peft_lora_inference_20_1.png)
 
+### Customize adapters strength
+For even more customization, you can control how strongly the adapter affects each part of the pipeline. For this, pass a dictionary with the control strengths (called "scales") to [`~diffusers.loaders.UNet2DConditionLoadersMixin.set_adapters`].
```

Comment:
> It's still unclear to me how this feature would allow me to control the output. I know I would be able to adjust the strength of Lora on the block level, but how would it correspond to the change in output? Are there any general rules you can follow?

You're basically imposing a granular level of control as to how you want to apply the adapter weights corresponding to the base layers. Think of applying a single adapter `scale` but applied on a more granular level. So, we control how much of the "adapter-effect" we want to see in the final generated output.

Reply:
Sometimes people come with some theories about how it affects the result but none of those are true in my tests. I use this kind of precision for when I use multiple LoRAs and when I want to use one that is too strong or even with overfitted ones.

The variation in the result depends on the prompt so the only way of knowing this is just testing, for this, people in automatic1111 use the xyz plot to know which combination works best.

In the sd-webui extension they show this image: https://user-images.githubusercontent.com/55250869/270830887-dff65f45-823a-4dbd-94c5-34d37c84a84f.jpg and as you can see there's no bad generation but just different variations.

Once this PR is merged, I plan to do more extensive testing and post what I learn.